Sources/ScipioKit/DependencyGraph.swift

```diff
@@ -0,0 +1,188 @@
+import Foundation
```

Comment:
Could you move this and `WeakReference` implementations to `ScipioKit/DependencyGraph/DependencyGraph.swift`